app/models/criteria/completed_task.rb

```diff
@@ -8,11 +8,16 @@ def self.create(searcher_id, assignee_id: nil, from_date: nil, to_date: nil, tag
         c.add_condition(Conditions::CompletedOnTo.create(to_date))
         c.add_condition(Conditions::Tags.create(tag_words))
         c.add_condition(Conditions::Completed)
+
         c.set_preparation do |plans|
           unless plans.planned_inner_join?(:taggings)
             plans.add(taggings: :left_outer)
           end
         end
+
+        c.set_sorter do |result_set|
+          result_set.sort_by { |r| r.completed_at }
```

Comment:
__[rubocop]__
* Style/SymbolProc: Pass `&:completed_at` as an argument to `sort_by` instead of a block.